services/frontend-service/src/ui/utils/store.tsx

```diff
@@ -253,3 +258,6 @@ export const useCurrentlyDeployedAt = (application: string, version: number) =>
             )
             .map((e) => e.name)
     );
+
+// Get current release dialog
+export const useCurrentReleaseDialog = () => useReleaseDialog(({ app, version }) => [app, version]);
```

Comment:
This is not quite true, you need to get the release data from the overview store, which contains commitMsg,locks,envs...etc

Reply:
Where should I store the information about which app/version is displayed now or not? I added a second call to `useCurrentlyDeployedAt` to fetch the envs